module2/src/com/epam/decomposition/GeometricalCalculator.java

```diff
@@ -0,0 +1,51 @@
+package com.epam.decomposition;
+
+public class GeometricalCalculator {
+    //task 3
+    public static double findHexagonArea(double sizeHexagon) {
+        return calculateEquilateralTriangleArea(sizeHexagon) * 6;
```

Comment:
please tell me what is six here. Hint: I want the constant NUMBER_OF_ANGLES = 6;